docs/source/1.0/spec/aws/aws-json-1_0-protocol.rst

```diff
@@ -61,4 +61,208 @@ See
     This protocol only permits the :ref:`httpPayload-trait` to be applied to
     members that target structures, documents, strings, blobs, or unions.
 
-*TODO: Add specifications, protocol examples, etc.*
+
+----------------
+Supported traits
+----------------
+
+The ``awsJson1_0`` protocol supports the following traits
+that affect serialization:
+
+.. list-table::
+    :header-rows: 1
+    :widths: 20 80
+
+    * - Trait
+      - Description
+    * - :ref:`jsonName <jsonName-trait>`
```

Comment:
Does this need to have `@cors` listed?

I think this supports `@timestampFormat`, even if services don't use it yet.